src/com/company/lesson1/Homework1.java

```diff
@@ -0,0 +1,14 @@
+package com.company.lesson1;
+
+public class Homework1 {
+    public static void main(String[] args) {
+        task1();
+
+
+    }
+
+    public static void task1() {
+        System.out.println("task1 run");
```

Comment:
Надо с воскл знаком:
`System.out.println("task1 run!");`
```suggestion
        System.out.println("task1 run!");
```